tagsbox/react-instantsearch/README.md

```diff
@@ -0,0 +1,118 @@
+# TagsBox Widget
+
+🎥  **[See it Live on CodeSandbox](https://codesandbox.io/s/vyvy30vyy)**
+
+## Demo
+
+![Demo of TagsBox Widget](https://cl.ly/6ea705021fda/Screen%252520Recording%2525202018-10-26%252520at%25252001.35%252520PM.gif)
+
+## Description
+
+This is the `TagsBox` Labs widget for [React InstantSearch](https://community.algolia.com/react-instantsearch/). You can use this widget to search and select tags with Algolia and use them later on for any purpose want.
+
+This widget displays an input that will offer tag suggestions while user is typing. Tags can be then clicked to be added to the input.
+
+This helps user to select records for another purpose than searching through them but to reuse them to complete another action later (like choosing a contact for sending an email or a category while creating a blog post for instance).
+
+**Table of Contents**
+
+* [Get the code](#get-the-code)
+  * [JavaScript](#javascript)
+  * [CSS](#css)
+* [Usage](#usage)
+  * [Requirements](#requirements)
+* [API](#api)
+  * [relevanceWidget(opts)](#relevancewidgetopts)
+* [Examples](#examples)
+* [Implementation details](#implementation-details)
+* [Contributing](#contributing)
+
+## Get the code
+
+This widget comes with JavaScript but also pre-defined CSS.
+
+### JavaScript
+
+You can copy and paste the JavaScript code from the repository itself, grab it in the [src/lib](src/lib) folder and use the `<Tag />` component.
+
+### CSS
+
+You can copy and paste the necessary CSS code from the repository itself, grab it in [src/lib/Tags.css](src/lib/Tags.css).
+
+## Usage
+
+The simplest usage is:
+
+```jsx
+import { InstantSearch } from 'react-instantsearch-dom';
+import TagsBox from 'instansearch-tagsbox-react';
+
+export const TagSelectedComponent = ({ hit }) => (
+    <Fragment>
+        {hit.lastname}
+    </Fragment>
+);
+
+export const TagSuggestionComponent = ({ hit }) => (
+    <Fragment>
+        {hit.firstname} {hit.lastname}
+    </Fragment>
+);
+
+<InstantSearch
+    appId="..."
+    apiKey="..."
+    indexName="..."
+>
+    <TagsBox
+        tagSelectedComponent={TagSelectedComponent}
+        tagSuggestionComponent={TagSuggestionComponent}
+        onUpdate={(newTags, oldTags) => console.log}
```

Comment:
The `console.log` wouldn't work like that. I would rename `oldTags` to `previousTags`.

```suggestion
        onUpdate={(newTags, previousTags) => console.log(newTags, previousTags)}
```